packages/server/utils/predicates.ts

```diff
@@ -1 +1,2 @@
-export const isNotNull = <T>(item: T | null): item is T => item !== null
+export const isNotNull = <T>(item: T | null | undefined): item is T =>
```

Comment:
we should probably rename this to `nullish` but that can wait for another time. LGTM